### ISSUE TITLE
JIT does not work 

### ISSUE THREAD

**Issue by chucamphong** [tailwindlabs/tailwindcss-jit#165]
### What version of @tailwindcss/jit are you using?

v0.1.12

### What version of Node.js are you using?

v15.11.0

### What browser are you using?

Microsoft Edge

### What operating system are you using?

Ubuntu 20.04

### Reproduction repository

https://www.youtube.com/watch?v=3O_3X7InOw8&t=713s

App.css:
```css
@import 'tailwindcss/base';
@import 'tailwindcss/components';
@import 'tailwindcss/utilities';
```

Log:
```error
ERROR in ./resources/css/app.css
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/postcss-loader/dist/cjs.js):
RangeError: Maximum call stack size exceeded
    at candidatePermutations (/var/www/html/node_modules/@tailwindcss/jit/src/lib/generateRules.js:19:32)
    at candidatePermutations.next (<anonymous>)
    at candidatePermutations (/var/www/html/node_modules/@tailwindcss/jit/src/lib/generateRules.js:41:10)
    at candidatePermutations.next (<anonymous>)
    at candidatePermutations (/var/www/html/node_modules/@tailwindcss/jit/src/lib/generateRules.js:41:10)
    at candidatePermutations.next (<anonymous>)
    at candidatePermutations (/var/www/html/node_modules/@tailwindcss/jit/src/lib/generateRules.js:41:10)
    at candidatePermutations.next (<anonymous>)
    at candidatePermutations (/var/www/html/node_modules/@tailwindcss/jit/src/lib/generateRules.js:41:10)
    at candidatePermutations.next (<anonymous>)
    at processResult (/var/www/html/node_modules/webpack/lib/NormalModule.js:675:19)
    at /var/www/html/node_modules/webpack/lib/NormalModule.js:774:5
    at /var/www/html/node_modules/loader-runner/lib/LoaderRunner.js:399:11
    at /var/www/html/node_modules/loader-runner/lib/LoaderRunner.js:251:18
    at context.callback (/var/www/html/node_modules/loader-runner/lib/LoaderRunner.js:124:13)
    at Object.loader (/var/www/html/node_modules/postcss-loader/dist/index.js:104:7)

1 ERROR in child compilations (Use 'stats.children: true' resp. '--stats-children' for more details)
webpack compiled with 2 errors
```

**Comment by munjalpatel** [comment]
I am also facing the same exact issue with `nuxt` and `tailwind`

**Comment by fedeci** [comment]
Seems to be similar to #166

**Comment by kgnfth** [comment]
same with nuxt tailwind

```
 ERROR  Failed to compile with 1 errors


 ERROR  in ./node_modules/@nuxtjs/tailwindcss/dist/runtime/tailwind.css

Module build failed (from ./node_modules/@nuxt/postcss8/node_modules/postcss-loader/dist/cjs.js):
RangeError: Maximum call stack size exceeded
    at candidatePermutations (/home/deejavu/nuxtravolta/node_modules/@tailwindcss/jit/src/lib/generateRules.js:19:32)
    at candidatePermutations (/home/deejavu/nuxtravolta/node_modules/@tailwindcss/jit/src/lib/generateRules.js:41:10)
    at candidatePermutations.next (<anonymous>)
    at candidatePermutations (/home/deejavu/nuxtravolta/node_modules/@tailwindcss/jit/src/lib/generateRules.js:41:10)
    at candidatePermutations.next (<anonymous>)
    at candidatePermutations (/home/deejavu/nuxtravolta/node_modules/@tailwindcss/jit/src/lib/generateRules.js:41:10)
    at candidatePermutations.next (<anonymous>)
    at candidatePermutations (/home/deejavu/nuxtravolta/node_modules/@tailwindcss/jit/src/lib/generateRules.js:41:10)
    at candidatePermutations.next (<anonymous>)
    at candidatePermutations (/home/deejavu/nuxtravolta/node_modules/@tailwindcss/jit/src/lib/generateRules.js:41:10)
    at candidatePermutations.next (<anonymous>)
    at candidatePermutations (/home/deejavu/nuxtravolta/node_modules/@tailwindcss/jit/src/lib/generateRules.js:41:10)
    at candidatePermutations.next (<anonymous>)
    at candidatePermutations (/home/deejavu/nuxtravolta/node_modules/@tailwindcss/jit/src/lib/generateRules.js:41:10)
    at candidatePermutations.next (<anonymous>)
    at candidatePermutations (/home/deejavu/nuxtravolta/node_modules/@tailwindcss/jit/src/lib/generateRules.js:41:10)

 @ ./node_modules/@nuxtjs/tailwindcss/dist/runtime/tailwind.css 4:14-207 15:3-20:5 16:22-215
 @ ./.nuxt/App.js
 @ ./.nuxt/index.js
 @ ./.nuxt/client.js
 @ multi ./node_modules/eventsource-polyfill/dist/browserify-eventsource.js (webpack)-hot-middleware/client.js?reload=true&timeout=30000&ansiColors=&overlayStyles=&path=%2F__webpack_hmr%2Fclient&name=client ./.nuxt/client.js

ℹ Waiting for file changes
ℹ Memory usage: 252 MB (RSS: 544 MB)
ℹ Listening on: http://localhost:3000/
```

**Comment by fedeci** [comment]
I am working on this btw.

**Comment by kgnfth** [comment]
> I am working on this btw.

Thx it working.

**Comment by adamwathan** [comment]
Fixed in v01.13, sorry about that!
